src/types/sdk.ts

```diff
@@ -68,6 +73,9 @@ export interface SdkOptions extends FunctionalConfigurations {
   steps?: Array<StepTypes | StepConfig>
   enterpriseFeatures?: EnterpriseFeatures
   customUI?: UICustomizationOptions | null
+  crossDeviceClientIntro?: CrossDeviceClientIntroOptions
```

Comment:
Are we accepting both flattened and nested options for this feature? 🤔

Reply:
No, we're not. I was initially considering implementing it as a nested option but decided against it. Missed this when cleaning it up.